daq/host.py

```diff
@@ -102,7 +102,8 @@ def __init__(self, runner, device, config):
         self._no_test = self.config.get('no_test', False)
         self.device = device
         self.target_mac = device.mac
-        self.target_port = device.port.port_no
+        self.target_port = device.port.port_no or device.nonof_port_num
```

Comment:
this still doesn't seem right -- why is "device.port.port_no" not always set correctly?

Reply:
device.port.port_no is used to denote port triggered tests in [daq/runner.py.](https://github.com/anurag6/daq/blob/59e9ba44a37790c9fea294a2beb59229da6e4ed1/daq/runner.py#L715). Setting it there causes all sorts of issues becase of that assumption.  I am not sure how to remove that assumption without causing significant issues.